templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -568,6 +568,16 @@
 			{{end}}
 		{{end}}
 
+		<div class="ui divider"></div>
+		<div class="ui reference equal width compact grid">
+			<div class="row">
+				<span class="text column">References: {{.Issue.ReferenceLink}}</span>
```

Comment:
This needs to be i18n-ed. Luckily that's quite easy to achieve. As this is your first contributing, a little help guide:

We first go into [`options/locale/locale_en-US.ini`](https://github.com/go-gitea/gitea/blob/main/options/locale/locale_en-US.ini). Don't get scared of it's length. 

Now we find a nice place to add a new translation key. I propose to add it to the existing "repo.issue". So placing it at line [L1483](https://github.com/go-gitea/gitea/blob/main/options/locale/locale_en-US.ini#L1483) should do the job. I think a key like `issue.reference_link` should do(Yes I left out the `repo` part, because that's the name of the section).

Now this is interesting, we need to handle dynamic content(the reference link). So that's a bit more work to achieve.

The first part is luckily the same:
`issue.reference_link = References: `
But to add the dynamic content, we add `%s`, so it becomes:
`issue.reference_link = References: %s`

Wow! That's the first part of allowing this string to be translated.
Now we need to say here in the template, Hey! Go lookup this translation key in the user's choice of language and oh here is the argument that will be taken into the translated string.

```suggestion
				<span class="text column">{{.i18n.Tr "repo.issues.reference_link" .Issue.ReferenceLink}}</span>
```

And voila you're now i18n-compliant with this string.

Reply:
Awesome! Yes, I agree - I was prioritising Go/CSS stuff rather than i18n-compat. Flawed perspective, but if I make contributions in future, I'll make sure there's an i18n translation string.